Source/settings/maps/Pallet Town.js

```diff
@@ -8,6 +8,10 @@ FullScreenPokemon.FullScreenPokemon.settings.maps.library["Pallet Town"] = {
             "direction": 2,
             "push": true
         },
+        "Fly Location": {
```

Comment:
I'm not a fan of `"Fly Location"` as a Location name (seems redundant). Maybe `"HM Transport"` or something like that?